src/WP_CLI/Fetchers/Theme.php

```diff
@@ -19,13 +19,39 @@ class Theme extends Base {
 	 * @return object|false
 	 */
 	public function get( $name ) {
-		$theme = wp_get_theme( $name );
-
-		if ( ! $theme->exists() ) {
+		// Workaround to equalize folder naming conventions across Win/Mac/Linux
+		// Returns false if theme stylesheet doesn't exactly match existing themes.
+		$existing_themes      = wp_get_themes( array( 'errors' => null ) );
+		$existing_stylesheets = array_keys( $existing_themes );
+		if ( ! in_array( $name, $existing_stylesheets, true ) ) {
+			$inexact_match = $this->find_inexact_match( $name, $existing_themes );
+			if ( false !== $inexact_match ) {
+				$this->msg .= sprintf( " Did you mean '%s'?", $inexact_match );
+			}
 			return false;
 		}
 
+		$theme = $existing_themes[ $name ];
+
 		return $theme;
 	}
+
+	/**
+	 * Find and return the key in $existing_themes that matches $name with
+	 * a case insensitive string comparison.
+	 *
+	 * @param string $name Name of theme received by command.
+	 * @param array  $existing_themes Key/value pair of existing themes, key is
+	 *               a case sensitive name.
```

Comment:
Alignment is off here:
```suggestion
	 *                                a case sensitive name.
```